linkerd/protocol/h2/src/main/scala/io/buoyant/linkerd/protocol/H2Config.scala

```diff
@@ -21,8 +21,10 @@ import io.buoyant.config.PolymorphicConfig
 import io.buoyant.linkerd.protocol.h2.{H2ClassifierConfig, H2RequestAuthorizerConfig}
 import io.buoyant.router.h2.ClassifiedRetries.{BufferSize, ClassificationTimeout}
 import io.buoyant.router.h2.{ClassifiedRetryFilter, DupRequest}
+import io.buoyant.router.http.ForwardClientCertFilter
 import io.buoyant.router.{ClassifiedRetries, H2, RoutingFactory}
 import io.netty.handler.ssl.ApplicationProtocolNames
+
```

Comment:
linkerd style is to have all the imports grouped together (no blank line) and sorted alphabetically.

Reply:
(this comment applies to other files in this review as well)